modules/ROOT/pages/prerequisites/index.adoc

```diff
@@ -234,9 +252,16 @@ It is strongly recommend to use a reverse proxy like {traefik-url}[Traefik], {ng
 
 == Filesystems and Shared Storage
 
-In addition to well known metadata like _name_, _size_ and _mtime_ (time a file was last modified), oCIS allows users to add arbitrary metadata like _tags_ and _comments_. Therefore oCIS requires and works with POSIX-compliant file systems where this metadata can be mapped to extended attributes. S3 storage is supported too, but requires POSIX-compliant local storage for metadata persistance.
+oCIS currently supports two different internal filesystem drivers which are `ocisfs` and `s3ng`.
+
+* When the `ocisfs` driver is used, data and metadata must be on a posix compliant filesystem.
+* When the `s3ng` driver is used, data resides on a S3 bucket and the metadata will be stored on a posix compliant filesystem which needs to be extra provisioned. This is necessary for performance reasons.
+
+Other drivers can be used too like for Ceph or EOS filesystem, but no support can be given because they are not developed or maintained by ownCloud. 
```

Comment:
```suggestion
Other drivers can be used too like for the Ceph or EOS filesystem, but no support can be given because they are not developed or maintained by ownCloud. 
```